azure/v14.0.0/release.yaml

```diff
@@ -0,0 +1,63 @@
+# Generated with:
+# devctl release create --name 14.0.0 --base v13.1.1 --releases . --provider azure --overwrite true --component kubernetes@1.19.4 --component app-operator@3.0.0 --component cluster-operator@0.23.20 --component azure-operator@5.2.1-dev --component containerlinux@2605.10.0 --component etcd@3.4.14 --app cluster-autoscaler@1.19.1 --app azure-scheduled-events@0.1.0 --app cert-exporter@1.6.0
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v14.0.0
+spec:
+  apps:
+    - name: cert-exporter
+      version: 1.6.0
+    - name: chart-operator
+      version: 2.6.0
+    - componentVersion: 1.6.5
```

Comment:
CoreDNS is here and in components, should only appear once. I think this was already and issue in earlier releases.

Reply:
removed